scripts/mobile-snapshot-diff.py

```
@@ -0,0 +1,193 @@
+    if args.suite:
+        run_suite()
+        return
+
+    if args.send:
+        send_input(args.send if args.send.endswith("\n") else args.send + "\n")
+        time.sleep(0.6)
+
+    if args.watch:
+        try:
+            while True:
+                os.system("clear")
```

Ruff (0.15.14)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)